### PR TITLE
#52 update readmes

### PR DESCRIPTION
**Pull request by qrafiq1**
Added some spice.

### CODE REVIEW

**Review by qrafiq1 — COMMENTED** [review]
Nioce

File: archive/README.MD [hunk 1]
```diff
@@ -40,7 +40,7 @@ pip3 install -r requirements.txt
     - Logs the status of operations to facilitate debugging.
     - Contains a lambda_handler function for AWS Lambda integration.
 
-## Secrets Management 🕵️‍♂️
+## Secrets Management 🕵🏽‍♂️
```
Comment (SurinaCS) [inline archive/README.MD:43]: really representative!

**Review by SurinaCS — APPROVED** [review]

**Review by keoghrmj — APPROVED** [review]
love the changes xd

**Review by FarihaChoudhury — COMMENTED** [review]
Thanks for implementing our changes

**Pull request merged**